crates/typst-library/src/loading/read.rs

```diff
@@ -25,13 +25,13 @@ use crate::World;
 #[func]
 pub fn read(
     engine: &mut Engine,
-    /// Path to a file.
+    /// ファイルへのパス。
```

Comment:
```suggestion
    /// ファイルのパス。
```

個人的にはこちらの方が自然な気がします。

Reply:
特に拘りはないですが、どちらかといえば「ファイルのパス」を支持します。

Reply:
皆様、ご意見ありがとうございます！
CBOR等のページも含めて、「ファイルのパス」に表現を統一させていただきます。